src/elk-monitoring/pagopa/paywallet/dashboards/walletDashboard.ndjson

```diff
@@ -1,2 +1,110 @@
-{"excludedObjects":[],"excludedObjectsCount":0,"exportedCount":1,"missingRefCount":0,"missingReferences":[]}
\ No newline at end of file
+{
+  "attributes": {
+    "description": "",
+    "kibanaSavedObjectMeta": {
+      "searchSourceJSON": "{\"query\":{\"query\":\"\",\"language\":\"kuery\"},\"filter\":[]}"
+    },
+    "optionsJSON": "{\"useMargins\":true,\"syncColors\":false,\"syncCursor\":true,\"syncTooltips\":false,\"hidePanelTitles\":false}",
```

Comment:
inline json in this file since this is a [ndjson](https://github.com/ndjson/ndjson-spec) file where each json is separated by a new line (take the kibana export api call raw output)

Reply:
Done in [e245efe](https://github.com/pagopa/pagopa-infra/pull/2645/commits/e245efe73e2c868326e9b29ba8e4fc0a982304e2)